docs: Update access list guide docs (#66733)
gravitational/teleport: docs/pages/identity-governance/access-lists/guide.mdx

Author: https://lisa-guided-al-doc.d3b94eevwi10ji.amplifyapp.com/docs/identity-governance/access-lists/guide/

{/* lint disable page-structure remark-lint */}

This guide walks you through the Teleport Web UI to:

- Create an Access List
- Assign a member to it
- Verify permissions granted through the list membership

> 

Reviewer: I would point users here where they should look if they want to create the list with IaC. I know we mention it in next steps as well but it's all the way down so it might be worth saying "this guide walks you through web UI but if you want to use IaC follow this guide instead".

When the guided setup asks you to add labels, add at least one — for example,
`env: test`. We'll use this label in Step 3 to select the host. (You can also
match all hosts with a `*` wildcard, but using a real label is closer to how
you'd build an Access List in practice.)
</Admonition>

The rest of this guide assumes you've enrolled an SSH server.

## Step 2/4. Create a test user

We need to create a simple test user that has only the `requester` role, which has no default access
to anything within a cluster. This user will only be used for the purposes of this guide, so you may use
another user if you so choose. If you would rather use your own user, skip to the next step.

From the sidebar menu, click **Zero Trust Access** > **Users** > **Enroll Users**. Fill in `test-user` as
the name and select `requester` as the role.

![Create Test User](../../../img/access-controls/access-lists/create-test-user.png)

Submit the form to create the user, then navigate to the provided URL in order to set up the credentials for your test user.
Try logging into the cluster with the test user to verify that no resources show up in the resources page.

## Step 3/4. Create an Access List

> Next, we'll create a simple Access List that grants its members access to the SSH

Reviewer: ```suggestion
Next, we'll create an Access List that grants its members access to the SSH
```

host we enrolled in Step 1. Login as the admin user mentioned in the
prerequisites.

From the sidebar menu, hover over **Add New** then click **Access List**.

Enter `test-list` as the Access List name, optionally add a description, then
select **Standing Access Guide** — this is the flow that grants members access on
sign-in, which is what we want for this tutorial. (The other option,
[Just-in-Time Access](./jit-access-list.mdx), makes resources *requestable*
instead of granted automatically.)

Click **Start Guide**.

<Admonition type="tip" title="Prefer Terraform?">
As you fill out the guided flow, a Terraform panel on the side mirrors your input
live. At the final **Deployment** step you can choose to apply the list with Teleport
directly or follow the in-UI instructions to apply it with Terraform.
</Admonition>

### Walk through the guide

The guide will step you through six screens.

> 

Reviewer: Nit: This extra heading seems unnecessary.
```suggestion
```

#### 2. Define Resource Identities or Principals

Under "List Owners" select `editor` as a required role, then add your administrative user under "Add
Eligible List Owners." By selecting `editor` as a required role, this will ensure that any owner of the list
must have the `editor` role in order to actually manage the list. If the user loses this role later, they will
not be able to manage the list, though they will still be reflected as an owner.
- Under Server Identities **Logins**, add the OS user(s) you want members to be
  able to log in as on the host (for example, `ubuntu` on a Linux server or your
  local username on macOS).

![Select an owner](../../../img/access-controls/access-lists/select-owner.png)
Note: this screen only asks for SSH logins because Server was the only resource
type you selected in the previous step.

Under "Members" select `requester` as a required role, then add your test user to the Access List. Similar to
the owner requirements, this will ensure that any member of the list must have the `requester` role in order to
be granted the access described in this list. If the user loses this role later, they will not be granted the

> roles or traits described in the Access List.

Reviewer: I notice we removed these instructions for "custom" access list. Should we keep them but move to a separate guide instead? I think it's still useful.

Author: i created a separate custom guide: https://lisa-guided-al-doc.d3b94eevwi10ji.amplifyapp.com/docs/identity-governance/access-lists/custom-access-list/

> enrolled in Step 1 in the resource list, and should be able to SSH into it using

Reviewer: Nit: Maybe show how to SSH ("tsh ssh" command basically).

(!docs/pages/identity-governance/access-lists/includes/acl-permissions.mdx!)

   ```yaml
   app_service:
     enabled: true
     debug_app: true
   ```
## Step 1/4. Enroll a test resource

1. Restart Teleport:
We need at least one resource on the cluster to grant access to. If you already
have a resource enrolled that you can use for testing, skip to the next step.

   ```code
   $ sudo systemctl reload teleport
   ```
Otherwise, follow the <a href="/get-started/connect/" target="_blank" rel="noopener">Connect Infrastructure</a>
guide to enroll a host through the Web UI's guided setup. The linked page uses an
Ubuntu server as its example, but the same flow works for your own machine
(Linux or macOS) — just pick your OS during the guided setup and run the
install command it gives you.

The "dumper" app should show up in the resource list.
<Admonition type="tip">
When the guided setup asks you to add labels, add at least one — for example,
`env: test`. We'll use this label in Step 3 to select the host. (You can also
match all hosts with a `*` wildcard, but using a real label is closer to how
you'd build an Access List in practice.)
</Admonition>

> ![Debug app](../../../img/access-controls/access-lists/debug-app.png)

Reviewer 2: Can  the `debug-app.png` file be removed. After removed this is referenced anywhere
